wcfsetup/install/files/lib/data/user/notification/UserNotificationEditor.class.php

```diff
@@ -29,6 +30,8 @@ public function markAsConfirmed() {
 			'confirmTime' => TIME_NOW,
 			'mailNotified' => 1
 		]);
+
+		EventHandler::getInstance()->fireAction($this, 'markAsConfirmed');
```

Comment:
Is this method even used? Maybe this method should be deprecated, if not and no event should be added.

Reply:
Currently it's not used by the WSC itself - neither by my own packages nor others I know.
I'd appreciate a deprecation/removal using the new deprecation exceptions.

Reply:
The exception is postboned to WSC 3.2. But, there should not be the event. A plugin should use the "correct" API.

Reply:
Did you mean WSC 5.3? Because this PR is targeting WSC 5.2. ;)